style.css

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
+@import url('./Components/alert.css');
+@import url('./Components/avatar.css');
+@import url('./Components/button.css');
+@import url('./Components/card.css');
+@import url('./Components/img-responsive.css');
+@import url('./Components/form.css');
+@import url('./Components/badge.css');
+@import url('./docs/css/index.css');
+@import url('./Components/modal.css');
+@import url('./Components/navbar.css');
+@import url("./Components/toast.css");
+@import url('./Components/slider.css');
+@import url('./Components/grid.css');
+@import url('./Components/list.css');
+@import url('./Components/rating.css');
+@import url('./Components/typography.css');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
```

Comment:
Double import, in same file, can be removed

Reply:
Thanks, done.